java.d/src/main/java/org/firehol/netdata/plugin/configuration/EnvironmentConfigurationService.java

```diff
@@ -77,4 +110,30 @@ protected Path readNetdataConfigDir() throws EnvironmentConfigurationException {
 		return configDir;
 
 	}
+
+	/**
+	 * Checks whether all given environment Netdata tracing flags are set.
+	 * 
+	 * @see <a href=
+	 *      "https://github.com/firehol/netdata/wiki/Tracing-Options">Tracing
+	 *      Options</a>
+	 */
+	public boolean isDebugFlagSet(long flags) {
+		if (getDebugFlags() == null)
```

Comment:
I personally do not like `if ... else` statements without braces. This can introduce bugs more likely than with braces.